packages/common/src/visualizations/types/index.ts

```diff
@@ -33,6 +33,19 @@ export enum VizIndexType {
     CATEGORY = 'category',
 }
 
+export function getColumnAxisType(column: VizColumn): VizIndexType {
```

Comment:
is this the same as `getVizIndexTypeFromDimensionType`?

Reply:
Annoyingly similar, but this is for the generic type instead of the Semantic runner one. I'd like to merge these.